### ISSUE TITLE
RFE: improve logs readability

### ISSUE THREAD

**Issue by delabassee** [fnproject/fdk-java#162]
I'm hitting an issue with Fn Flow. The logs are a bit hard to parse and could easily be made a bit more readable.

```
time="2018-10-12T15:53:18Z" level=debug msg="An error occurred in function: com.fnproject.fn.api.flow.FunctionInvokeFailedException: Failed to call function\n" action="server.handleFnInvokeCall)-fm" app_id=01CJGZBEMKNG8G00GZJ0000003 call_id=01CSMFPSFSNG8G00GZJ000000V fnID=01CS1HZT5GNG8G00GZJ000000F fn_id=01CS1HZT5GNG8G00GZJ000000F id=01CSMFPSFSNG8G00GZJ000000V image="trip:0.0.227" user_log=true```
-> `fnID` and `fn_id`are dupes; `call_id` and `id` too.

```'time="2018-10-12T15:53:18Z" level=debug msg="com.fnproject.fn.api.flow.FunctionInvokeFailedException: Failed to call function\n" action="server.handleFnInvokeCall)-fm" app_id=01CJGZBEMKNG8G00GZJ0000003 call_id=01CSMFPSFSNG8G00GZJ000000V fnID=01CS1HZT5GNG8G00GZJ000000F fn_id=01CS1HZT5GNG8G00GZJ000000F id=01CSMFPSFSNG8G00GZJ000000V image="trip:0.0.227" user_log=true'```
-> `server.handleFnInvokeCall)-fm`-> ')-fm' looks strange.

And at the end, I still don't know why the function couldn't be called ;-)

**Comment by zootalures** [comment]
Those logs are from Functions not the FDK - the FDK just emits 
```
An error occurred in function: com.fnproject.fn.api.flow.FunctionInvokeFailedException: Failed to call function
```
